src/components/navigation/navigation-item/navigation-item.component.ts

```diff
@@ -0,0 +1,138 @@
+import {
+    Component,
+    Input,
+    ViewEncapsulation,
+    ContentChildren,
+    QueryList,
+    Renderer2,
+    ElementRef,
+    Optional,
+    SkipSelf,
+    ContentChild,
+    Output,
+    AfterViewInit,
+  HostListener
+} from '@angular/core';
+import { Router, RouterLinkActive, NavigationEnd, RouterState, ActivatedRoute, UrlTree } from '@angular/router';
```

Comment:
RouterLinkActive, RouterState and ActivatedRoute can be removed as they are never used

Reply:
Imports removed.